### PR TITLE
Homework 4

### PR DESCRIPTION
**Pull request by PashaMisan**
Packer

### CODE REVIEW

File: packer/scripts/backup.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+max_dump_count=7
+region="eu-central-1"
```
Comment (saaverdo) [inline packer/scripts/backup.sh:3]: region value can be parametrized.
As the script is used in timer, you can define env file in `service` unit just as you did for app.service

File: packer/db.pkr.hcl [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    destination = "/home/ubuntu/backup.sh"
+  }
+  provisioner "shell" {
+    inline = [
+      "sudo apt update",
+      "sudo apt install -y mariadb-server awscli"
+    ]
+    pause_before = "10s"
+  }
+  provisioner "shell" {
```
Comment (saaverdo) [inline packer/db.pkr.hcl:49]: If you need to pass packer variables to script you can use `environment_vars` section:
```
  provisioner "shell" {
    environment_vars = [
      "REGION=${var.region}"
    ]
    ...
```

but if you want to run script with `sudo` it'll be a bit tricky:
```
  provisioner "shell" {
    environment_vars = [
      "REGION=${var.region}"
    ]
    script = "./scripts/example.sh"
    execute_command = "sudo bash -c '{{.Vars}} {{.Path}}'"
  }
```
  
execute_command = "sudo bash -c '{{.Vars}} {{.Path}}'"

**Review by saaverdo — APPROVED** [review]
Nice solution. Using systemd timer - good.
Also respect for description.
Good work!
Approved )

File: packer/db.pkr.hcl (hunk 2 above)
Comment (PashaMisan) [reply packer/db.pkr.hcl:49]: Understood, I'll remember that.Thanks for the comment!

**Pull request merged**